tests/virt_autotest/esxi_open_vm_tools.pm

```diff
@@ -101,9 +122,15 @@ sub wait_for_vm_network {
     }
 
     # Monitor the networking after bring up/down the VM
-    while ($times && console('svirt')->run_cmd($cmd)) {
+    do {
+        if (is_svirt) {
+            $is_nw_up = console('svirt')->run_cmd($cmd);
+        }
+        elsif (is_qemu) {
+            $is_nw_up = script_run($cmd);
+        }
         $times--;
-    }
+    } while ($times && $is_nw_up);
```

Comment:
Any follow-up actions after this "monitor" block ?

Reply:
Not quite understand. This block will be repeatedly called as per the test scenario. The only change in this function is that added a condition to judge whether qemu or svirt backend.